### PR TITLE
Adding marginalia tool

### LINKED ISSUE #52: Marginalia_'Transcribe Marginalia' button
This will have the same transcription functionality as for text, but people draw a square/rec. around the area. 

Also, please make it so that the border is a dark line but the centre is clear.

### PR DESCRIPTION
**Pull request by simoneduca**
Fixes #52

### CODE REVIEW

File: app/modules/transcribe/annotations/graphic.directive.js [hunk 1]
```diff
@@ -4,23 +4,23 @@ var _ = require('lodash');
...
         restrict: 'A',
         scope: {
             data: '='
         },
-        templateUrl: 'annotations/image.html',
+        templateUrl: 'annotations/graphic.html',
     };
     return directive;
 
     // @ngInject
```
Comment (rogerhutchings) [inline app/modules/transcribe/annotations/graphic.directive.js]: no need for `// @ngInject` here

Comment (rogerhutchings) [inline app/modules/transcribe/annotations/graphic.directive.js]: no need for `// @ngInject` here

File: app/modules/transcribe/annotations/marginalia.directive.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        replace: true,
+        restrict: 'A',
+        scope: {
+            data: '='
+        },
+        templateUrl: 'annotations/marginalia.html',
+    };
+    return directive;
+
+    // @ngInject
```
Comment (rogerhutchings) [inline app/modules/transcribe/annotations/marginalia.directive.js]: no need for `// @ngInject` here

File: app/modules/transcribe/annotations/marginalia.html [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<g class="marginalia-annotation">
+    <rect class="marginalia" draggable="data" ng-attr-x="{{ data.x }}" ng-attr-y="{{ data.y }}" ng-attr-height="{{ data.height }}" ng-attr-width="{{ data.width }}"></rect>
```
Comment (rogerhutchings) [inline app/modules/transcribe/annotations/marginalia.html]: With this number of attributes, I'd put one on each line

Comment (rogerhutchings) [inline app/modules/transcribe/annotations/marginalia.html]: I've seen this in a few places; separate lines after a certain number of attributes makes it easier to maintain.

Comment (simoneduca) [inline app/modules/transcribe/annotations/marginalia.html]: It's Beautify on Brackets that does the formatting. I'll try to change it.

**Pull request merged**